### ISSUE TITLE
just realized

### ISSUE THREAD

**Issue by ChillerDragon** [chichilku/chichilku3#10]
this is probably the issuelessest issue i ever made:

The server class variables are different for every player because they have their own thread.
Well thats nothing too new to me so using global variables for global event packets or stuff like that isn't possible. Thats why i am sending the connection packets every x ticks and not on player join....

buuuut! if the class vars are useless how does the correct id assignment is working?
Well i don't really know and i guess you the future me while reading the issue can take care of.
I assume that the @current_id just works because its the same everywhere and the clients save their own ids locally but when you decrement it the value might differ or idk? Investigating the class vars in threads might be the key for events like player death (event #9) and disconnection (freeing slots #7)

**Comment by ChillerDragon** [comment]
tldr lmao

**Comment by ChillerDragon** [comment]
Fixed thread madness in https://github.com/chichilku/chichilku3/commit/f11f9cc9199d95bcf5121b10f201c6b806d1007c
